sdks/java/extensions/euphoria/src/main/java/org/apache/beam/sdk/extensions/euphoria/core/client/operator/FlatMap.java

```diff
@@ -160,11 +160,7 @@ public static OfBuilder named(@Nullable String name) {
    * Last builder in a chain. It concludes this operators creation by calling {@link
    * #output(OutputHint...)}.
    */
-  public interface OutputBuilder<OutputT> extends Builders.Output<OutputT> {
-
-    @Override
-    Dataset<OutputT> output(OutputHint... outputHints);
-  }
+  public interface OutputBuilder<OutputT> extends Builders.Output<OutputT> {}
```

Comment:
Is this interface still necessary?

Reply:
It is not 👍 I've removed the extra interface where possible.